RELEASENOTES.md

```diff
@@ -15,6 +14,27 @@ release) was only available in one of the two variants, we note that
 in the release notes.
 
 
+# v1.7.2-aws release notes
+This release requires [Libfabric v1.11.0](https://github.com/ofiwg/libfabric/releases/tag/v1.11.0)
+or later and supports [NCCL v2.18.1-1](https://github.com/NVIDIA/nccl/releases/tag/v2.18.1-1) while
```

Comment:
I think much of our testing has been `v2.18.3-1`.

Reply:
Yeah, fair.  Updated.